examples/research_projects/dreambooth_inpaint/requirements.txt

```diff
@@ -0,0 +1,7 @@
+diffusers==0.9.0
```

Comment:
@thedarkzeno I think this needs to be 0.10.2 as 0.9.0 doesn't have check_min_version